clients/drcachesim/common/options.cpp

```diff
@@ -640,3 +640,11 @@ droption_t<bool> op_enable_drstatecmp(
     DROPTION_SCOPE_CLIENT, "enable_drstatecmp", false, "Enable the drstatecmp library.",
     "When true, this option enables the drstatecmp library that performs state "
     "comparisons to detect instrumentation-induced bugs due to state clobbering.");
+
+#ifdef BUILD_PT_TRACER
+droption_t<bool> op_enable_kernel_tracing(
```

Comment:
Please add entries to api/docs/release.dox on this option (and on the drpttrace extension library if that was not added before).  Label as still experimental.

Reply:
Done.